runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -202,6 +203,18 @@ std::unique_ptr<onert::ir::Model> loadModel(const std::string filename,
   return std::unique_ptr<onert::ir::Model>(nullptr);
 }
 
+std::unique_ptr<onert::ir::train::TrainingInfo>
+loadTrainingInfo(const std::shared_ptr<onert::ir::Model> &model)
+{
+  const auto tinfo_name = onert::train::traininfo_loader::TRAININFO_METADATA_NAME;
```

Comment:
For a while, I considered this `tinfo_name` should be `const auto &` not `const auto`. Soon I found out the `TRAINING_METADATA_NAME` is a char pointer and realized that there is no need to use `&`. In order to enhance the code readability, it would be a good solution to use the `std::string` instead of `char pointer`. I made a PR about my suggestion #12424. This is just my suggestion and I don't force it. :)

Reply:
Thank you for the suggestion and the draft PR! :smile: 
I like the way you suggested - it improves readability and efficiency. 

But as you know, I already fixed traininfo_loader once :sweat_smile: - https://github.com/Samsung/ONE/pull/12410. 
And  I'd like to close this issue - https://github.com/Samsung/ONE/issues/11692#issuecomment-1855114192 as soon as possible. (After this PR, one last thing is to remove the default argument from onert_train.) 

So, If you don't mind I'd like to merge this PR as it is and... fix it to use std::string instead of char* in the next PR.

Reply:
My suggestion requires several changes except for this PR. So I want to merge it as it is. :)
If you don't mind, I can apply my suggestion after merged this PR with other reviewer's approval.

Reply:
> If you don't mind, I can apply my suggestion after merged this PR with other reviewer's approval.

I appreciate it :grinning: Thank you.